Date: Mon, 12 Feb 2018 22:29:56 +0000
Subject: drivers: introduce AMAZON_DRIVER_UPDATES

This provides a central place to maintain out-of-tree drivers.
Renamed from VENDOR_AMAZON because the name was no longer appropriate.

Signed-off-by: Munehisa Kamata <kamatam@amazon.com>
Reviewed-by: Cristian Gafton <gafton@amazon.com>
Reviewed-by: Guru Anbalagane <guruanb@amazon.com>

Reviewed-by: Eduardo Valentin <eduval@amazon.com>
Reviewed-by: Anchal Agarwal <anchalag@amazon.com>
Signed-off-by: Vallish Vaidyeshwara <vallish@amazon.com>

diff --git a/drivers/Kconfig b/drivers/Kconfig
--- a/drivers/Kconfig
+++ b/drivers/Kconfig
@@ -239,4 +239,5 @@ source "drivers/peci/Kconfig"
 
 source "drivers/hte/Kconfig"
 
+source "drivers/amazon/Kconfig"
 endmenu
diff --git a/drivers/Makefile b/drivers/Makefile
--- a/drivers/Makefile
+++ b/drivers/Makefile
@@ -189,3 +189,4 @@ obj-$(CONFIG_COUNTER)		+= counter/
 obj-$(CONFIG_MOST)		+= most/
 obj-$(CONFIG_PECI)		+= peci/
 obj-$(CONFIG_HTE)		+= hte/
+obj-$(CONFIG_AMAZON_DRIVER_UPDATES)    += amazon/
diff --git a/drivers/amazon/Kconfig b/drivers/amazon/Kconfig
new file mode 100644
--- /dev/null
+++ b/drivers/amazon/Kconfig
@@ -0,0 +1,15 @@
+#
+# Amazon driver updates configuration
+#
+
+config AMAZON_DRIVER_UPDATES
+	bool "Amazon Driver Updates"
+	default y
+	depends on PCI || EXPERIMENTAL
+	---help---
+	 Amazon driver updates includes out-of-tree drivers and/or modifeid
+	 versions of the drivers present in the stable kernel tree.
+
+if AMAZON_DRIVER_UPDATES
+
+endif # AMAZON_DRIVER_UPDATES
diff --git a/drivers/amazon/Makefile b/drivers/amazon/Makefile
new file mode 100644
--- /dev/null
+++ b/drivers/amazon/Makefile
@@ -0,0 +1,3 @@
+#
+# Amazon Driver Updates
+#
